subjects/road_intersection/README.md

```diff
@@ -32,15 +32,15 @@ _________________________|_______________________
            r ↓           |
 _______________          |          _____________
                |         | ←  ↑  → |
-               |    ↓    | r  s  l |
+               |    ↓    | l  s  r |
                |    ↓    |    ↑    |
                |    ↓    |    ↑    |
                |    ↓    |    ↑    |
                |    ↓    |    ↑    |
                |    ↓    |    ↑    |
 ```
 
-For clarification reasons we will assume that a lane can have three different routes (consider you are in the vehicle position):
+For clarification reasons we will assume that a lane can have three different routes (consider that you are in the vehicle position):
```

Comment:
Does this mean that each direction (e.g. North->South) has three different routes?

The usage of the word 'lane' seems to contradict what is written on line 16:

https://github.com/01-edu/public/blob/332e922aaec357b2a8b5e9ecc25c8f929de95701/subjects/road_intersection/README.md?plain=1#L16

Reply:
Yes, each lane has the possibility to go three different ways.
i specified line 16 better

Reply:
We're using 2-lane highways. My comment is irrelevant.